### PR TITLE
TKO-80 -- FE: Create an attendance sheet for the admin

### PR DESCRIPTION
**Pull request by raimahossain**
# Description
Create an attendance sheet for the admin
  
# Testing
Go to /workshopview and see if the attendance sheet appears (including pagination)

### CODE REVIEW

**Review by andrewMETALAB — CHANGES_REQUESTED** [review]
I'm getting several console errors including:
`[Vue warn]: Property or method "user" is not defined on the instance but referenced during render...`
`TypeError: "_vm.user is undefined"`

Also getting a terminal error when running the watch command:
`Error: [BABEL] /Users/andrewgrano/websites/takeoff/resources/js/app.js: The `core-js` version must be false, 2 or 3, but got true...`

Are you *not* seeing these errors? If so, maybe me .env is out of date?

File: resources/js/router/views/WorkshopView/index.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<template>
+    <div class="container py-3">
+        <div>
+            <AttendanceCard/>
```
Comment (andrewMETALAB) [inline resources/js/router/views/WorkshopView/index.vue]: Raima, this line is what was producing the JS console error! Just remove this line and you'll be good to go. 👍

**Review by andrewMETALAB — CHANGES_REQUESTED** [review]
See my comment bellow 👍

**Review by andrewMETALAB — APPROVED** [review]

**Review by alecnicolas — APPROVED** [review]
y35

**Pull request merged**